src/main/java/duke/Parser.java

```diff
@@ -72,7 +76,9 @@ public static Command parse(String fullCommand) {
                     int indexFrom = fullCommand.indexOf("/from");
                     int indexTo = fullCommand.indexOf("/to");
                     if (indexFrom != -1 && indexTo != -1) {
-                        t = new Event(fullCommand.substring(6, indexFrom), fullCommand.substring(indexFrom + 6, indexTo - 1), fullCommand.substring(indexTo + 4));
```

Comment:
Good parsing practices but I also recommend using method `trim()` to remove the leading and trailing whitespace in the task description!